src/app/app.module.ts

```diff
@@ -1,11 +1,18 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
+import { TranslateModule, TranslateLoader } from '@ngx-translate/core';
```

Comment:
Let's separate the imports in three groups:

[angular imports]

[third party imports]

[application imports]

Also sort them alphabetically like it's explained here: https://angular.io/guide/styleguide#import-line-spacing